src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        WorkWithFile workWithFile =new WorkWithFile();
+
+        String fromFileName ="input.csv";
```

Comment:
Consider using a constant for the filename 'input.csv'. It will make the code more maintainable and the purpose of the file more clear.